includes/utils.php

```diff
@@ -329,14 +329,21 @@ public static function get_timezone_string() {
 	 * @static
 	 *
 	 * @param string $post_type Optional. Post type slug. Default is 'page'.
+	 * @param string|null $template_type Optional. Query arg 'template_type'. Default is null.
 	 *
 	 * @return string A URL for creating new post using Elementor.
 	 */
-	public static function get_create_new_post_url( $post_type = 'page' ) {
-		$new_post_url = add_query_arg( [
+	public static function get_create_new_post_url( $post_type = 'page', $template_type = null ) {
+		$query_args = [
```

Comment:
It should work with the `Documents_Manager`, please check with @Nevoss  / @iNewLegend if the method is deprecated.

Reply:
See https://github.com/elementor/elementor/pull/12550/files#diff-a4c20b6d7391a224faf11e6161c80b408e2a0f5a71a92dc11b7eafe3cad83f94R329

Reply:
It's not ready yet for merge